worldeditadditions_commands/commands/selectors/scube.lua

```diff
@@ -0,0 +1,55 @@
+-- ███████  ██████ ██    ██ ██████  ███████
+-- ██      ██      ██    ██ ██   ██ ██
+-- ███████ ██      ██    ██ ██████  █████
+--      ██ ██      ██    ██ ██   ██ ██
+-- ███████  ██████  ██████  ██████  ███████
+worldedit.register_command("scube", {
+	params = "[<axis1> [<axis2> [<axis3>]]] <length>",
+	description = "Set WorldEdit region position 2 at a set distance along 3 axes.",
+	privs = { worldedit = true },
+	require_pos = 1,
+	parse = function(params_text)
+		local wea, vec, tmp = worldeditadditions, vector.new(0, 0, 0), {}
```

Comment:
Same here with `tmp`. Could it be renamed to better reflect it's purpose at all?

Reply:
I could always call it dump. But in all seriousness if you think of a better name let me know.

Reply:
Ah, I see. It's somewhat difficult to track when we have multiple unstructured variables in a table like this. Could they be refactored to be individual variables at all?